<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>420</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>BPG Nino Mtavruli</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget  {background-color: #FFF7D4;}

QPushButton {
	border: 1px solid #C07F00;
	background: #FFF7D4;
	border-radius: 6px;

}

QPushButton:hover {
	border: 1px solid black;
	background-color: #FFD95A;
}
QLineEdit {
	border: 1px solid #C07F00;
	border-radius: 6px;	
	padding: 6px;
	background: white;
}

QLineEdit:focus {
	border: 1px solid #F8AD25;
}

QLineEdit:hover {
	background-color: #FFD95A;
}

QComboBox {
	border: 0.5px solid #C07F00;
}

QComboBox:hover {
	background-color: #FFD95A;
}
</string>
  </property>
  <widget class="QLineEdit" name="search_by_phone">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BPG Nino Mtavruli</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border-top-right-radius: 0px;
    border-bottom-right-radius: 0px;
	border-top-left-radius: 4px;
    border-bottom-left-radius: 4px;
}</string>
   </property>
   <property name="placeholderText">
    <string>ტელეფონი</string>
   </property>
  </widget>
  <widget class="QLabel" name="main_label">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>20</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BPG Nino Mtavruli</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>საფასურის გადახდა</string>
   </property>
  </widget>
  <widget class="QLabel" name="searchbyphon_label">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>60</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BPG Nino Mtavruli</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ძებნა ტელეფონის ნომრით</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fname">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>191</y>
     <width>170</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BPG Nino Mtavruli</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="placeholderText">
    <string>სახელი</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lname">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>190</y>
     <width>170</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BPG Nino Mtavruli</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="placeholderText">
    <string>გვარი</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>400</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="payment_method">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>270</y>
     <width>170</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BPG Nino Mtavruli</family>
     <pointsize>12</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;BPG Nino Mtavruli&quot;;</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="amount">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>270</y>
     <width>170</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BPG Nino Mtavruli</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>თანხა / წუთი</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pay_button">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>354</y>
     <width>152</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BPG Nino Mtavruli</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>გადახდა</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-top-right-radius: 4px;
    border-bottom-right-radius: 4px;
	border-top-left-radius: 0px;
    border-bottom-left-radius: 0px;
}</string>
   </property>
   <property name="text">
    <string>🔍</string>
   </property>
  </widget>
  <widget class="QLabel" name="if_not_in_db">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>144</y>
     <width>341</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BPG Nino Mtavruli</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>თუ კლიენტი არაა ბაზაში შეავსეთ მონაცემები</string>
   </property>
  </widget>
  <widget class="QLabel" name="warning">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>167</y>
     <width>282</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BPG Nino Mtavruli</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;ყველა ველის შევსება სავალდებულოა!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>230</y>
     <width>170</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BPG Nino Mtavruli</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="placeholderText">
    <string>ტელეფონი</string>
   </property>
  </widget>
  <widget class="QLabel" name="balanc_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>157</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BPG Nino Mtavruli</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>მიმდინარე ბალანსი: </string>
   </property>
  </widget>
  <widget class="QLabel" name="balance">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BPG Nino Mtavruli</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="category">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>230</y>
     <width>170</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BPG Nino Mtavruli</family>
     <pointsize>12</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;BPG Nino Mtavruli&quot;;</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>search_by_phone</tabstop>
  <tabstop>search_button</tabstop>
  <tabstop>fname</tabstop>
  <tabstop>lname</tabstop>
  <tabstop>phone</tabstop>
  <tabstop>category</tabstop>
  <tabstop>payment_method</tabstop>
  <tabstop>amount</tabstop>
  <tabstop>pay_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
